apps/api/src/app/events/usecases/parse-event-request/parse-event-request.usecase.ts

```diff
@@ -188,8 +197,26 @@ export class ParseEventRequest {
     transactionId,
     discoveredWorkflow?: DiscoverWorkflowOutput | null
   ) {
-    const jobData: IWorkflowDataDto = {
+    const commandArgs = {
       ...command,
+    };
+
+    if ('to' in commandArgs) {
+      const sanitizedSubscribers = this.sanitizeRecipients(commandArgs.to);
```

Comment:
@BiswaViraj please combine this do with a single reduce function, there is no need to iterate on this twice.